### PR TITLE
new eks-d releases

### PR DESCRIPTION
**Pull request by jaxesn**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by TerryHowe — APPROVED** [review]
/lgtm

File: EKSD_LATEST_RELEASES [hunk 1]
```diff
@@ -3,15 +3,15 @@ releases:
   number: 17
   kubeVersion: v1.18.20
```
Comment (TerryHowe) [inline EKSD_LATEST_RELEASES:4]: Going to get rid of 18 here?

Comment (jaxesn) [reply EKSD_LATEST_RELEASES:4]: I think we actually need to keep 18 because 19 uses the 18 pause image

**Pull request merged**